WcaOnRails/app/webpacker/components/UserAvatar.js

```diff
@@ -8,10 +8,12 @@ function UserAvatar({
   avatarClass = '',
   title = '',
   size = 'medium',
+  disableHover = false,
+  breakCache = false,
 }) {
   const { url, thumb_url: thumbUrl, thumb } = avatar;
   // The avatar thumbnail url is at thumb_url for officers but at thumb.url for team members.
-  const thumbnailUrl = thumbUrl || thumb.url || url;
+  const thumbnailUrl = `${thumbUrl || thumb.url || url}${breakCache ? `?${Date.now() / 1000}` : ''}`;
```

Comment:
I also feel it does the same thing. It was using breakCache for WcaOnRails/app/views/users/edit_avatar_thumbnail.html.erb, hence I did the same. I don't know why exactly this is used, but I am assuming there can be case where people edits thumbnail immediately after changing their DP, and if thumbnail doesn't break cache, it might show old image and might cause some issues.

Reply:
Okay :+1: